gauge.go

```diff
@@ -159,7 +159,7 @@ func main() {
 			track.Format()
 			formatter.FormatSpecFilesIn(*specFilesToFormat)
 		} else if *validate {
-			track.Validation()
+			track.Validation(*validate)
```

Comment:
Why pass `validate` flag? This will be `true` always. Instead it should be `false` since the `hide-suggestion` flag does not exist in legacy usage.